### ISSUE TITLE
[CLI] Error getting releaseGold contract info

### ISSUE THREAD

**Issue by jcortejoso** [celo-org/celo-monorepo#6459]
### Expected Behavior

Celocli should return the releaseGold contract information when running `celocli releasegold:show --contract <release_gold_contract_address>`.

### Current Behavior

Next error is returned:
```
╰─ celocli releasegold:show --contract 0xf671BAcb43179F30F3fDD58F81367Fd5C33AC1F5                                          
    Error: value out of range (argument="value", value=20, code=INVALID_ARGUMENT, version=bytes/5.0.9)
    Code: INVALID_ARGUMENT
```

### Environment
Celocli version: @celo/celocli@1.0.0-beta
Node version: Tested with v10.23.0 and v12.13.1

It works with celocli version @celo/celocli@0.0.60

**Comment by barbaraliau** [comment]
@jcortejoso Can you update to the latest version `1.0.3`? I believe this should work now.

I tested with the command you showed:

```sh
celocli releasegold:show --contract 0xf671BAcb43179F30F3fDD58F81367Fd5C33AC1F5

releaseGoldWrapperAddress: 0xf671BAcb43179F30F3fDD58F81367Fd5C33AC1F5
beneficiary: 0xd449DB2DAba9388266B78B5D40a4c40D72de36d0
authorizedSigners:
  voter: 0x389Ed6D8048CeFEc51B6E8AF74eCE15958B962fF
  validator: 0xf671BAcb43179F30F3fDD58F81367Fd5C33AC1F5
  attestations: 0xf671BAcb43179F30F3fDD58F81367Fd5C33AC1F5
releaseOwner: 0x0a2B8C88952B57ca0a84928fa27339f9690FF134
refundAddress: 0x0000000000000000000000000000000000000000
liquidityProvisionMet: true
canValidate: true
canVote: true
releaseSchedule:
  releaseStartTime: Fri Jun 05 2020 01:00:00 GMT-0600
  releaseCliff: Sat Jun 05 2021 01:00:00 GMT-0600
  numReleasePeriods: 1
  releasePeriod: 1 hour
  amountReleasedPerPeriod: 19999000000000000000000 (~2.000e+22)
isRevoked: false
revokedStateData:
  revocable: false
  canExpire: true
  releasedBalanceAtRevoke: 0
  revokeTime: 0
balanceStateData:
  totalWithdrawn: 0
  maxDistribution: 115792089237316195423570985008687907853269984665640564039457584007913129639935 (~1.158e+77)
  totalBalance: 19998001173752803563426 (~2.000e+22)
  remainingTotalBalance: 19998001173752803563426 (~2.000e+22)
  remainingUnlockedBalance: 19897000000000000000000 (~1.990e+22)
  remainingLockedBalance: 101001173752803563426 (~1.010e+20)
  currentReleasedTotalAmount: 0
```

**Comment by barbaraliau** [comment]
Fixed in new versions `>1.0.3`
